MAUI/DataGrid/grid-events.md

```diff
@@ -172,6 +251,20 @@ private void dataGrid_CellExited(object sender, DataGridCellExitedEventArgs e)
 {% endhighlight %}
 {% endtabs %}
 
+### CellExited Command and its Parameter
+When the cell is exited, the `CellExitedCommand`triggered that binds to the command which you have defined in your ViewModel. With this we can also use the `CellExitedCommandParameter` to be passed to the `Command` when it is executed.
```

Comment:
The command will be invoked when the mouse pointer exits a cell in the SfDataGrid. The instance of `DataGridCellExitedEventArgs` will be passed as the default value of the command's parameter. It can be customized by setting the `CellExitedCommandParameter` property.

Reply:
Changed